### ISSUE TITLE
Feature request: `repeat / multiply`

### ISSUE THREAD

**Issue by vorillaz** [plexis-js/plexis#48]
Repeats the input text number of times.

```javascript
@param {String} [text] The input text.
@returns {Integer} times How many times shall we repeat the text, default to 1
@param {String} glue The glue for the output, default to '' 
@returns {String} The repeated string
```

**Example usage**

```javascript
import repeat from '@plexis/repeat';

repeat('ABCD');
// => true

repeat('Apollo11', 2);
// => 'Apollo11Apollo11'

repeat('x', 2);
// => 'xx'

repeat('x', 2, '+');
// => 'x+x'

repeat('xx', 2, '_');
// => 'xx_xx'
```

**Aliases**

```javascript
import repeat from '@plexis/repeat';
import {repeat, multiply} from 'plexis';
```

**Comment by lazav94** [comment]
I will take this one.

**Comment by vorillaz** [comment]
Implemented by #51
